### PR TITLE
Add 'pod tag' commands.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: src/cli/cmds/pod.toit [hunk 1]
```diff
@@ -130,6 +130,90 @@ create-pod-commands -> List:
...
+        """
+  cmd.add tag-cmd
+
+  tag-add-cmd := Command "add"
+      --help="""
+        Add tags to uploaded pods.
+
+        The pods to change are specified through a pod reference like
+        name@tag or name#revision.
+        If only a pod name is provided, the pod with the 'latest' tags is
```
Comment (kasperl) [inline src/cli/cmds/pod.toit:145]:
```suggestion
        If only a pod name is provided, the pod with the 'latest' tag is
```

File: src/cli/cmds/pod.toit [hunk 2]
```diff
@@ -130,6 +130,90 @@ create-pod-commands -> List:
...
+      --help="""
+        Change the tags of an uploaded pod.
+        """
+  cmd.add tag-cmd
+
+  tag-add-cmd := Command "add"
+      --help="""
+        Add tags to uploaded pods.
+
+        The pods to change are specified through a pod reference like
```
Comment (kasperl) [inline src/cli/cmds/pod.toit:143]: Can this change the tags of multiple pods? A pod reference specifies a single pod, no?

I guess you can pass multiple pod references here, but tags/pods/pod references is in this section are somewhat confusing.

Comment (kasperl) [inline src/cli/cmds/pod.toit:143]:
```suggestion
        The pods to change are specified through pod references like
```

File: src/cli/cmds/pod.toit [hunk 3]
```diff
@@ -130,6 +130,90 @@ create-pod-commands -> List:
...
+            --help="A tag to add to the pod."
+            --multi,
+        Flag "force"
+            --short-name="f"
+            --help="Force tags even if they already exist."
+            --default=false,
+      ]
+      --rest=[
+        Option "reference"
+            --help="A pod reference: a UUID, name@tag, or name#revision."
```
Comment (kasperl) [inline src/cli/cmds/pod.toit:166]: There can be more than one. Not sure how you usually write the help for such cases.

File: src/cli/cmds/pod.toit [hunk 4]
```diff
@@ -130,6 +130,90 @@ create-pod-commands -> List:
...
+        If '--force' is provided, tags are added even if they already exists, moving
+        the tags to the selected pods.
+
+        Note: this operation is not atomic. It downloads the existing tags, adds the
+        new tags, and then sets the combined tags.
+        """
+      --options=[
+        Option "tag"
+            --short-name="t"
+            --help="A tag to add to the pod."
```
Comment (kasperl) [inline src/cli/cmds/pod.toit:157]: There can be more than one. Not sure how to write the help for such cases.

File: src/cli/cmds/pod.toit [hunk 5]
```diff
@@ -130,6 +130,90 @@ create-pod-commands -> List:
...
+      ]
+      --run=:: tag-add it
+  tag-cmd.add tag-add-cmd
+
+  tag-remove-cmd := Command "remove"
+      --aliases=["delete"]
+      --help="""
+        Remove tags from pods.
+
+        The pods to change are specified by their pod-name, and not
```
Comment (kasperl) [inline src/cli/cmds/pod.toit:191]:
```suggestion
        The pods to change are specified by their pod names, and not
```

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/cli/cmds/pod.toit (hunk 2 above)
Comment (floitsch) [reply src/cli/cmds/pod.toit]: done.

File: src/cli/cmds/pod.toit (hunk 1 above)
Comment (floitsch) [reply src/cli/cmds/pod.toit]: done.

File: src/cli/cmds/pod.toit (hunk 4 above)
Comment (floitsch) [reply src/cli/cmds/pod.toit:157]: I think the "multi" flag is there to make it clear that they can appear multiple times.

File: src/cli/cmds/pod.toit (hunk 3 above)
Comment (floitsch) [reply src/cli/cmds/pod.toit:166]: Same.

File: src/cli/cmds/pod.toit (hunk 5 above)
Comment (floitsch) [reply src/cli/cmds/pod.toit]: done.

File: src/cli/cmds/pod.toit (hunk 2 above)
Comment (floitsch) [reply src/cli/cmds/pod.toit]: I reworded it a bit.

**Pull request merged**